src/Cache/RedisFactory.php

```diff
@@ -73,6 +74,16 @@ private function normalizeServer(string $server): array
             $parsedServer['password'] = urldecode($pass);
         }
 
+        $database = $parsedServer['path'] ?? null;
+        unset($parsedServer['path']);
+
+        if ($database !== null) {
+            /**
+             * @todo Validate this is an integer and throw an exception if not for the next major version release
+             */
```

Comment:
Probably one line is enough.

```suggestion
            // TODO For the next major version, validate this is an integer and throw an exception otherwise
```